test/NuGet.Core.Tests/NuGet.Common.Test/MigrationRunnerTests.cs

```diff
@@ -62,5 +62,35 @@ public void Run_WhenExecutedInParallelThenOnlyOneMigrationFileIsCreated_Success(
             Assert.Equal(1, files.Length);
             Assert.Equal(Path.Combine(directory, "1"), files[0]);
         }
+
+        [Fact]
+        public void Run_WhenAThreadAbandonsMutexThenNextMigrationRunReleasesMutexAndCreatesMigrationFile_Success()
+        {
+            Mutex _orphan = new Mutex(false, "NuGet-Migrations");
+
+            // Arrange
+            Thread t = new Thread(new ThreadStart(AbandonMutex));
+            t.Start();
+            t.Join();
+
+            string directory = MigrationRunner.GetMigrationsDirectory();
+            if (Directory.Exists(directory))
+                Directory.Delete(path: directory, recursive: true);
```

Comment:
I prefer using braces:

```suggestion
            if (Directory.Exists(directory))
            {
                Directory.Delete(path: directory, recursive: true);
            }
```

Reply:
I somehow prefer not to add braces for this case.  https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md suggests `Braces may be omitted only if the body of every block associated with an if/else if/.../else compound statement is placed on a single line.`